src/Contracts/Provider.php

```diff
@@ -7,7 +7,7 @@ interface Provider
     /**
      * Redirect the user to the authentication page for the provider.
      *
-     * @return \Illuminate\Http\RedirectResponse
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Illuminate\Http\RedirectResponse
```

Comment:
@taylorotwell , not to complain, but `\Illuminate\Http\RedirectResponse` already extends `\Symfony\Component\HttpFoundation\RedirectResponse`. Why put both of them here?! :)